rest/handler.go

```diff
@@ -99,6 +99,7 @@ type handler struct {
 	runOffline            bool
 	queryValues           url.Values // Copy of results of rq.URL.Query()
 	permissionsResults    map[string]bool
+	dbAuthStringFunc      func(bodyJSON []byte) (string, error)
```

Comment:
I feel like this would be better as just 'authScopeFunc' - omit the db, and make it clear it's expected to return an authScope string.

I also prefer to define an alias for for function signature - it doesn't make that much of a difference here, but it's a lot clearer to read when being used as a parameter (as below).